sklearn/pipeline.py

```diff
@@ -20,12 +21,74 @@
 from .externals import six
 from .utils import tosequence
 from .utils.metaestimators import if_delegate_has_method
-from .externals.six import iteritems
 
 __all__ = ['Pipeline', 'FeatureUnion']
 
 
-class Pipeline(BaseEstimator):
+class _BasePipeline(BaseEstimator):
```

Comment:
Of course. Just goes to show how much review this has had over the past 3 years...